com.unity.render-pipelines.universal/Runtime/2D/Passes/Utility/RendererLighting.cs

```diff
@@ -161,8 +161,8 @@ private static bool RenderLightSet(IRenderPass2D pass, RenderingData renderingDa
 
                     renderedAnyLight = true;
 
-                    if (light.lightType == Light2D.LightType.Sprite && light.lightCookieSprite != null && light.lightCookieSprite.texture != null)
-                        cmd.SetGlobalTexture(CookieTexID, light.lightCookieSprite.texture);
+                    if (light.lightType == Light2D.LightType.Sprite && light.spriteLightCookie != null && light.spriteLightCookie.texture != null)
```

Comment:
these null checks are every where.... maybe we should do an `isValid` to not repeat these things.

Reply:
also we should really ask our selves, when would sprite.Texture be null...

Reply:
I couldn't figure out how to make this null, so I did both suggestions